tools/issue-labeler/eng/Versions.props

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Any reason these aren't just in the csproj?

Reply:
I see you are trying to share this with all the csprojs. If that is the case I suggest creating a Directory.Build.props in the root of your folder which will automatically get included.

Reply:
An example of the props file that Wes mentioned:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Directory.Build.props